### PR TITLE
Changed transformation to fix

### LINKED ISSUE #165: Rename `transformation` to `fix`

### PR DESCRIPTION
**Pull request by Deeeeksha**
#165

### CODE REVIEW

File: src/core.ts [hunk 1]
```diff
@@ -34,7 +34,7 @@ function generateResultsToUndo(
         if (!resultOfLint) throw new Error(`The result of ${resultToFix.filePath} is not found.`);
         // NOTE: THIS IS HACK.
         // Usually, the result with `output` property does not contain `source` property.
-        // However, `source` property is required for undoTransformation.
+        // However, `source` property is required for undofix.
```
Comment (mizdra) [inline src/core.ts:37]: `undoTransformation` is the name of a method that was originally implemented in the Core class, but the method no longer exists (it was removed during the refactoring process).

It makes sense to change this sentence, but the name of the method named `undofix` does not exist as well. Could you please change it to the following?

```suggestion
        // However, `source` property is required to undo.
```

Comment (Deeeeksha) [reply src/core.ts]: @mizdra can you please check now.

**Comment by mizdra** [comment]
Thank you for contributing!

I intended #165 to include not only code comment changes, but also anything that would cause destructive changes, such as changing the type or method name that is being exported.

So this PR does not solve #165 by itself, but it helps to solve it. I will try to merge it.

File: src/core.ts [hunk 2]
```diff
@@ -34,7 +34,7 @@ function generateResultsToUndo(
         if (!resultOfLint) throw new Error(`The result of ${resultToFix.filePath} is not found.`);
         // NOTE: THIS IS HACK.
         // Usually, the result with `output` property does not contain `source` property.
-        // However, `source` property is required for undoTransformation.
+        // However, `source` property is required for undo.
```
Comment (mizdra) [inline src/core.ts:37]: I want it to be `to`!

```suggestion
        // However, `source` property is required to undo.
```

**Review by mizdra — APPROVED** [review]
LGTM！

**Pull request merged**